### PR TITLE
TAJO-1991: Tablespace::getVolume should take filter predication.

### PR DESCRIPTION
**Pull request by hyunsik**
See https://issues.apache.org/jira/browse/TAJO-1991.

This patch depends on TAJO-2007.

### CODE REVIEW

File: tajo-plan/src/main/java/org/apache/tajo/plan/rewrite/TableStatUpdateRewriter.java [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+      try {
+        if (table.getStats() != null) {
+          return storage.getTableVolumn(table.getUri(), filter);
+        }
+      } catch (UnsupportedException t) {
+        LOG.warn(table.getName() + " does not support Tablespace::getTableVolume()");
+        // By default, return 1GB to avoid a single task
+      }
+
+      return StorageUnit.GB;
```
Comment (jinossy) [inline tajo-plan/src/main/java/org/apache/tajo/plan/rewrite/TableStatUpdateRewriter.java]: How about the configurable value ?

**Comment by hyunsik** [comment]
I recover the default 1GB to the original value -1. Actually, it was related to many other code parts. It should be resolved in another jira.

**Comment by hyunsik** [comment]
I rebased and fixed all tests. It is ready to review.

**Comment by hyunsik** [comment]
Rebased.

**Comment by jihoonson** [comment]
This patch looks good to me. Test failure seems not related to the changes in this PR.

**Comment by hyunsik** [comment]
Rebased. Thanks!

**Comment by jihoonson** [comment]
+1 ship it!

**Pull request closed without merging**